public/assets/js/ability-scores.js

```diff
@@ -0,0 +1,98 @@
+// Retrieve selected race ability bonus info
```

Comment:
This is the work Christiana did on her rev of the project. Merged here because assuming it will be merged back up to hers and wanted to make sure it was stable.